src/custom-elements/src/components/popover/popover.tsx

```diff
@@ -404,6 +409,7 @@ export class Popover {
   }
 
   render() {
+    const closeButton = this.closable ? <chi-button class="-position--absolute" size="sm" onClick={() => this.hide()} type="close" /> : null;
```

Comment:
`-position--absolute` can be removed as it's defined in the CSS

```suggestion
    const closeButton = this.closable ? <chi-button size="sm" onClick={() => this.hide()} type="close" /> : null;
```